app/services/audit_log_service.rb

```diff
@@ -0,0 +1,7 @@
+class AuditLogService
+  include Callable
+
+  def self.call(message)
```

Comment:
```suggestion
  def call(message)
```

The `Callable` concern should handle this transparently.